lib/fastlane_core/devices.rb

```diff
@@ -1,14 +1,19 @@
 require 'open3'
 
 module FastlaneCore
-  class Simulator
+  class Devices
     class << self
-      def requested_os_type
-        'iOS'
+      def all(requested_os_type)
+        all_devices = simulators(requested_os_type)
+        if requested_os_type == "iOS"
+          all_devices += connected_devices
```

Comment:
I am not totally sure I am reading this correctly, but it looks like regardless of the type of connected device, it is being added too the `all_devices` array if the selected os is iOS. I'm not sure this is totally safe as one can also connect a tvOS device to their machine as well.